python/dgl/data/lrgb.py

```diff
@@ -0,0 +1,237 @@
+import hashlib
+import os
+import pickle
+
+import pandas as pd
+from ogb.utils import smiles2graph
+from tqdm import tqdm
+
+from .. import backend as F
+
+from ..convert import graph as dgl_graph
+from .dgl_dataset import DGLDataset
+from .utils import download, load_graphs, save_graphs, Subset
+
+
+class PeptidesStructuralDataset(DGLDataset):
+    r"""Peptides structure dataset for the graph classification task.
+
```

Comment:
please cite the corresponding original work like other datasets: https://docs.dgl.ai/api/python/dgl.data.html

Reply:
fixed